src/Input.jsx

```diff
@@ -56,18 +58,23 @@ const Input = React.createClass({
 
   render() {
     const placeholder = this.props.placeholder || '';
-    let size = this.props.value.length === 0 ?
-      placeholder.length :
-      this.props.value.length;
+    let extraProps = {};
+
+    if (this.props.inputAutoSize){
+        extraProps["size"] = (this.props.value.length === 0 ?
```

Comment:
since there are no more `extraProps` right now, and I don't know if there will be in the future, what do you think about moving this to a `size` variable which, by default is null and set it directly as `size={size}` (which won't be rendered when it's null)?